modules/integration/src/test/scala/scala/cli/integration/SemanticDbTestDefinitions.scala

```diff
@@ -44,7 +44,7 @@ trait SemanticDbTestDefinitions { _: CompileTestDefinitions =>
           .flatMap(opt => List("--javac-opt", opt))
         val javaSemDbOptions = Seq(
           "--javac-plugin",
-          "com.sourcegraph:semanticdb-javac:0.7.4",
+          "com.sourcegraph:semanticdb-javac:0.10.0",
```

Comment:
maybe we could pass this version from `deps.sc` via `Constants`?

Reply:
Done!